docs/docsite/rst/contributing/content_scoring.rst

```diff
@@ -14,10 +14,21 @@ the content's Community Score and Quality Score.
 Content Scoring is in early iterations, to be improved with
 user feedback and future additions to ansible-lint.
 
+Overall Score
```

Comment:
With this descriptive summary of the 'Overall Score', the sentence above in lines 12 and 13 can probably be removed.